frontend/components/suggestion-and-search.js

```diff
@@ -49,7 +49,7 @@ export default class SuggestionAndSearch extends React.Component {
       let suggestions
       try {
         const json = await res.json()
-        suggestions = json.badges
+        suggestions = json.badges || []
```

Comment:
This fixes the crash in the frontend when a response comes back without a `badges` array.